index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Documento</title>
```

Comment:
```suggestion
    <title>Document</title>
```

mira me parece que era mejor que diga document asi en ingles